Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -428,6 +428,7 @@ - (void)toxAV:(OCTToxAV *)toxAV callStateChanged:(OCTToxAVCallState)state friend
 
         if ((self.audioEngine.friendNumber == friendNumber) && [self.audioEngine isAudioRunning:nil]) {
             [self.audioEngine stopAudioFlow:nil];
+            [self.videoEngine stopSendingVideo];
```

Comment:
Can we have situation when audioEngine is not running, but videoEngine is? In that case it won't stop.